CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- Method `GET /:id` returns document containing only fields defined in the JSON Schema of the collection
```

Comment:
```suggestion
- Method `GET /:id` returns document containing only fields defined in the JSON Schema of the collection (in case environment variable `ENABLE_STRICT_OUTPUT_VALIDATION` is set to _true_.
```